src/openrct2/ride/TrackPaint.cpp

```diff
@@ -2167,6 +2168,20 @@ void track_paint(paint_session* session, Direction direction, int32_t height, co
             }
         }
 
+#ifdef __ENABLE_LIGHTFX__
+        if (lightfx_is_available())
+        {
+            uint8_t zOffset = 16;
```

Comment:
Couldn't you just use the base height and the clearance height of the tile.

Reply:
I use 16 because I wanted to lantern light to be coming out of the shop window. (And 23 for the toilets because that's where the little light bulb is)